mungegithub/Dockerfile

```diff
@@ -16,15 +16,15 @@ FROM google/debian:wheezy
 MAINTAINER Brendan Burns <bburns@google.com>
 RUN apt-get update
 RUN apt-get install -y -qq ca-certificates
-ADD mungegithub /mungegithub
-ADD blunderbuss.yml /blunderbuss.yml
 ADD path-label.txt /path-label.txt
 ADD generated-files.txt /generated-files.txt
+ADD blunderbuss.yml /blunderbuss.yml
 # User lists for submit-queue and 'needs-ok-to-merge'
 ADD committers.txt /committers.txt
 ADD whitelist.txt /whitelist.txt
 # Submit queue web interface
 ADD www /www
+ADD mungegithub /mungegithub
```

Comment:
I guess it slipped in, it means that I can build the container over and over and over changing only the binary and docker will cache all of the other stuff.  Putting 'mungegithub' at the top meant that it had to add /www when the mungegithub changed and it was slowing me down, man.